evap/staff/templates/staff_evaluation_textanswers_full.html

```diff
@@ -36,23 +36,27 @@
 
 {% block additional_javascript %}
     <script type="text/javascript">
-        function press(id, action) {
-            if($("#"+id+"-"+action).hasClass("active")) {
-                action = "unreview";
-            }
-            updateState(id, action);
-            var parameters = {"id": id, "action": action, "evaluation_id": {{ evaluation.id }}};
-            $.ajax({
-                type: "POST",
-                url: "{% url 'staff:evaluation_textanswers_update_publish' %}",
-                data: parameters,
-                error: function(){ window.alert("{% trans 'The server is not responding.' %}"); }
+        for (const form of document.getElementsByClassName("full-update-textanswer-form")) {
+            form.addEventListener("formdata", event => {
```

Comment:
I wonder if the UX was more intuitive (and easier to program) if we gave three buttons, forming a radio-based button group, instead (publish - unreviewed - hide).

Reply:
good idea > #1765